ibm_catalog.json

```diff
@@ -99,29 +107,25 @@
 						"diagrams": [
 							{
 								"diagram": {
-									"caption": "All inclusive Cloudability Onboarding",
+									"caption": "Standard Cloudability Enablement",
 									"type": "image/svg+xml",
 									"url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cloudability-onboarding/refs/heads/main/reference-architectures/cloudability-all-inclusive-onboarding.svg"
 								},
-								"description": "Billing reports are written to a Key Protect encrypted object storage bucket. IBM Cloudability is granted access to read from this bucket. Your IBM Cloud account is added to Cloudability so that it knows how to read the data from the object storage bucket."
+								"description": "Billing reports are written by IBM Cloud Billing to a Key Protect encrypted object storage bucket. IBM Cloudability is granted access to read the billing reports from this bucket which are updated every day. Your IBM Cloud account is added to Cloudability so that it knows is aware of the the object storage bucket to pull the billing data from. Optionally, monitoring and activity events are sent to IBM Cloud Monitoring and IBM Cloud Logs respectively to be able to audit access to the object storage bucket. For more details visit the [reference architecture documentation](/docs/track-spend-with-cloudability?topic=track-spend-with-cloudability-cloudability-enablement)"
```

Comment:
typo: Your IBM Cloud account is added to Cloudability so that it  ~~**knows**~~ is aware of the the object storage bucket to pull the billing data from.